Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN git clone https://github.com/benwebber/sqlite3-uuid.git
 WORKDIR /sqlite3-uuid
 RUN make
 
-FROM golang:1.17-alpine AS build-env
+FROM golang:1.22-alpine AS build-env
 
 RUN apk update && apk upgrade && \
```

Comment:
Ongerelateerd aan deze change maar verbaas mij een beetje over alle tools die geinstalleerd worden. Sowieso de vraag of alpine hier zo'n goede keuze is gezien de dependencies op native tooling (spatialite, etc), kan sneller tot issues leiden. Maar prima om nu zo te houden, want staat los van deze change.